community/bolt/src/docs/dev/messaging.asciidoc

```diff
@@ -129,6 +131,10 @@ In the case where a previous result stream has not yet been fully consumed, an a
 If an unacknowledged failure is pending from a previous exchange, the server will immediately respond with a single
 `IGNORED` message and take no further action.
 
+===== Response
+- `SUCCESS` if the statement has been accepted for execution
```

Comment:
Perhaps also mention the fields one would expect in this response? Ditto with `FAILURE`?